examples/03_datetime_encoder.py

```diff
@@ -120,8 +111,8 @@
 #
 # Here, for example, we want it to extract the day of the week.
```

Comment:
```suggestion
# Here, for example, we want it to extract the day of the week:
```

and I'll change for `default parameter with a new, custom instance.` above (can't put a suggestion there)